migrations/20220419172600-create-data_groups.js

```diff
@@ -1,18 +1,17 @@
-'use strict';
+"use strict";
 
 module.exports = {
-    async up(queryInterface, Sequelize) {
-
-        await queryInterface.sequelize.query(
-            `CREATE TABLE data_groups (
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(
+      `CREATE TABLE data_groups (
                 iddata_groups INT NOT NULL AUTO_INCREMENT,
                 title VARCHAR(45) NOT NULL,
+                hex_colour VARCHAR(7) DEFAULT NULL,
```

Comment:
Rather than editing this previous migration, we should add a new migration which ALTERs the table to add the new column (see other migrations for examples of how to do this).

We need to do it this way so that the migration can be run on the live databases, rather than having to recreate them and losing all existing data. Sequelize only keeps track of the name of migration files that have previously run.

Reply:
All done man, just give it check. 😁

Reply:
Nice👌